.github/workflows/release.yml

```diff
@@ -44,16 +42,13 @@ jobs:
       - name: Create Rpm Installer
         run: ./gradlew buildRpmPackage
```

Comment:
We should pass the `version` to this command and the `buildDebPackage` command as well, see https://stackoverflow.com/a/40239092